File: Pratiquer/split.js
function FindIntersection(strArr) { 
  
  let arr1 = strArr[0].split(',');
  console.log(arr1);
  let arr2 = strArr[1].split(',');
  console.log(arr2)
  let filteredArray = arr1.filter(value => arr2.includes(value));
  
  return filteredArray; 
}
   
console.log(FindIntersection(readline()));

File: split.py
def FindIntersection(strArr):

  intersect = []
  arr1 = strArr[0].split(',')
  print(arr1)
  arr2 = strArr[1].split(',')
  print(arr2)
  
  for i in arr1:
    if i in arr2:
      intersect.append(i)

  return intersect

print(FindIntersection(input()))
